RevenueCatUI/Purchasing/PurchaseHandler.swift

```diff
@@ -135,19 +131,10 @@ extension PurchaseHandler {
         self.restoredCustomerInfo = nil
         self.restoreError = nil
 
-<<<<<<< HEAD
-        let customerInfo = try await self.purchases.restorePurchases()
-        try await refreshSubscriptions()
-
-        withAnimation(Constants.defaultAnimation) {
-            self.restored = true
-            self.restoredCustomerInfo = customerInfo
```

Comment:
I know that in the comments it says that it's not set but before we were setting this information. Is it normal that now we don't do it anymore?

Reply:
Not sure why they did it, but they changed this part in the update. 
Even `var restored` is not present anymore